server/config/local.json-dist

```diff
@@ -27,5 +27,9 @@
   "metrics": {
     "sample_rate": 0
   },
-  "iframe_allowed_origins": ["http://127.0.0.1:8080"]
+  "iframe_allowed_origins": ["http://127.0.0.1:8080"],
+  "csp": {
```

Comment:
We are forcing the *default* to be true. But config files are evaluated subsequent to that,  so it can be overridden. Yes, setting it to true here is redundant. *shrug*